buildkite/aggregate_incompatible_flags_test_result.py

```diff
@@ -34,12 +34,20 @@
 
 REPO_PATTERN = re.compile(r"https?://github.com/(?P<owner>[^/]+)/(?P<repo>[^/]+).git")
 
+INCOMPATIBLE_FLAG_LINE_PATTERN = re.compile(
+    r"\s*(?P<flag>--incompatible_\S+)\s*(\(Bazel\s+(?P<version>[^:]+):\s+(?P<url>[^\)]+)\))?"
```

Comment:
I think this could be simplified using lazy quantifiers (`.+?`) as:

```
\s*(?P<flag>--incompatible_\S+) \(Bazel (?P<version>.+?): (?P<url>.+?)\)
```

However, I don't know if you on purpose used `\s*` instead of just ` ` when we expect a single whitespace character - are there cases where there's more than one?

Reply:
I designed the pattern so that it would match the current output, too (so that we still detect the flag, even if version and url are missing).

Reply:
What about this then?

```
\s*(?P<flag>--incompatible_\S+)\s*(\(Bazel (?P<version>.+?): (?P<url>.+?)\))?
```

Reply:
Done